04_Conditions.ipynb

```diff
@@ -147,33 +202,35 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "# Bound x to within a range\n",
-    "x = int(input(\"x? \"))\n",
-    "minimum_value = 0\n",
-    "maximum_value = 10\n",
-    "\n",
-    "if x < minimum_value:\n",
-    "    print(\"x too low\")\n",
-    "x = minimum_value\n",
-    "\n",
-    "if x > maximum_value:\n",
-    "    print(\"x is too high\")\n",
-    "x = maximum_value\n",
+    "def clamp_0_10(value):\n",
+    "    \"Bound value between 0 and 10 inclusive\"\n",
+    "    minimum_value = 0\n",
+    "    maximum_value = 10\n",
+    "    \n",
+    "    if value < minimum_value:\n",
+    "        print(f\"{value} is too small\")\n",
```

Comment:
Shouldn't it return minimum_value? Or is this a part of the task to identify it? :P